src/devices/cpu/t11/t11.h

```diff
@@ -106,6 +117,8 @@ class t11_device :  public cpu_device
 	PAIR                m_psw;
 	uint16_t            m_initial_pc;
 	uint8_t             m_wait_state;
+	int                 m_mcir;
+	uint16_t            m_vsel;
```

Comment:
It appears that `m_mcir` and `m_vsel` are supposed to remain valid when returning to the scheduler.  Should they be included in save states?

Also, things included in save states need to use explicitly sized types for portability (`int` is not explicitly sized).